.gitignore

```diff
@@ -3,3 +3,4 @@
 client/util/personal.py
 client/util/config.py
 server/linux_x11/config.py
+.idea
```

Comment:
It was intentional -- I'm not sure how you (or most people) prefer to deal with IDE-specific files, I've always found it easiest to just ignore htem all. Will pull that out if you'd prefer --